src/features/MoviesBrowser/MovieDetails/Details/styled.js

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 
 export const DetailsWrapper = styled.div`
-  max-width: 1368px;
+  max-width: 1920px;
```

Comment:
why did you increse the width. Tile should be smaller than website.

Reply:
Yes, I just wanted to check something with the image and I didn't change it to what it should be. An oversight :)